src/java.xml.crypto/share/classes/javax/xml/crypto/dsig/SignatureMethod.java

```diff
@@ -255,6 +255,22 @@ public interface SignatureMethod extends XMLStructure, AlgorithmMethod {
      */
     String RSA_PSS = "http://www.w3.org/2007/05/xmldsig-more#rsa-pss";
 
+    /**
+     * The <a href="http://www.w3.org/2021/04/xmldsig-more#eddsa-ed25519">
+     * ED25519</a> signature method algorithm URI.
```

Comment:
I think it would be beneficial to add a reference to the standards. Most of these URIs are defined in RFC 9131, but a few are from the W3C Recommendation. I suggest adding the following sentence to the class description:

"The signature method algorithm URIs defined in this class are specified in the [W3C Recommendation for XML-Signature Syntax and Processing](http://www.w3.org/TR/xmldsig-core/) and [RFC 9131](https://www.rfc-editor.org/info/rfc9131)."